website_event_questions_multiple/static/src/js/form_validation.js

```diff
@@ -0,0 +1,163 @@
+/** @odoo-module **/
+
+import publicWidget from "web.public.widget";
+import EventRegistrationForm from "website_event.website_event";
+
+import ajax from "web.ajax";
+
+/// declare widget
+// extends from EventRegistrationForm from ocb/addons/website_event/static/src/js/website_event.js
+export const EventRegistrationFormWithValidation = EventRegistrationForm.extend({
+    /// ------------------------------------------------
+    /// WARNING: code duplication for lack of extensibility
```

Comment:
If you copied code from elsewhere, you should add Copyright information in this file, author information in module manifest + change license if the one from where the code comes is not AGPL-3